Make convert to linear an export pass

mcr229 opened the pull request:
Summary: Let's remove the reliance on export pass from convert to linear pass. This allows us. We only use exported program in one place to find the bias.

Differential Revision: D62266927

swolchok reviewed (approved):
propagating internal accept to github

mcr229 committed:
Make convert to linear an export pass (#5133)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/5133

Let's remove the reliance on export pass from convert to linear pass. This allows us. We only use exported program in one place to find the bias.

Reviewed By: digantdesai

Differential Revision: D62266927
